recipe/move-level-zero-devel.sh

```diff
@@ -2,12 +2,12 @@
 
 set -ex
 
-pushd ${SRC_DIR}
```

Comment:
I did this specifically to avoid issues if conda changes the path from which it runs the build or move scripts for whatever reason. Also, scripts can be reused as-is which is nice. Is the change needed because scripts are somehow used for win and `pushd` is unavailable there? Otherwise, I think having absolute paths is generally better.

Reply:
Conda executes all build scripts from the same work directory. 

Even though `pushd` does exist on Windows, we should adhere to the common practice of using work directory for intermediate outputs. This work directory is the current directory when scripts begin execution.